api/webhooks/webhooks.py

```diff
@@ -101,9 +125,14 @@ def _call_webhook(
     return requests.post(str(webhook.url), data=json_data, headers=headers, timeout=10)
 
 
+@register_task_handler()
 def _call_webhook_email_on_error(
```

Comment:
If this is a task, I'd rather it was a public method.

Reply:
Changed it to a public method. Thanks